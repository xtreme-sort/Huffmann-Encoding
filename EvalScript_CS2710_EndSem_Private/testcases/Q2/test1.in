DeriveHuffmanTree
9
B 01000
C 110
V 100
Y 011
c 0101
d 01001
j 00
t 111
v 101
Quit
