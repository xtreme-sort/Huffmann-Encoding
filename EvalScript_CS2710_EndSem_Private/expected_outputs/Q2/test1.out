((j,(((B,d),c),Y)),((V,v),(C,t)));
